src/wp-includes/class-wp-block-type.php

```diff
@@ -204,6 +204,16 @@ class WP_Block_Type {
 	 */
 	public $style = null;
 
+	/**
+	 * Attributes supported by every block.
+	 *
+	 * @since 6.0.0
+	 * @var array
+	 */
+	const CORE_ATTRIBUTES = array(
```

Comment:
I couldn't decide between `CORE_ATTRIBUTES` and `BUILTIN_ATTRIBUTES`. I'm open to name suggestions :)

Reply:
> I couldn't decide between `CORE_ATTRIBUTES` and `BUILTIN_ATTRIBUTES`.

How about `GLOBAL_ATTRIBUTES` or `GLOBAL_BLOCK_ATTRIBUTES`?

Reply:
`GLOBAL_ATTRIBUTES` reads nice 👍🏻